static/app/components/events/autofix/autofixDrawer.tsx

```diff
@@ -0,0 +1,158 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import ProjectAvatar from 'sentry/components/avatar/projectAvatar';
+import {Breadcrumbs as NavigationBreadcrumbs} from 'sentry/components/breadcrumbs';
+import {Button} from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import AutofixFeedback from 'sentry/components/events/autofix/autofixFeedback';
+import AutofixMessageBox from 'sentry/components/events/autofix/autofixMessageBox';
+import {AutofixSteps} from 'sentry/components/events/autofix/autofixSteps';
+import {useAiAutofix} from 'sentry/components/events/autofix/useAutofix';
+import {DrawerBody, DrawerHeader} from 'sentry/components/globalDrawer/components';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
+import type {Group} from 'sentry/types/group';
+import type {Project} from 'sentry/types/project';
+import {getShortEventId} from 'sentry/utils/events';
+import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
+import {MIN_NAV_HEIGHT} from 'sentry/views/issueDetails/streamline/eventNavigation';
+
+interface AutofixDrawerProps {
+  event: Event;
+  group: Group;
+  project: Project;
+}
+
+export function AutofixDrawer({group, project, event}: AutofixDrawerProps) {
+  const {autofixData, triggerAutofix, reset} = useAiAutofix(group, event);
+  useRouteAnalyticsParams({
+    autofix_status: autofixData?.status ?? 'none',
+  });
+
+  const [_, setContainer] = useState<HTMLElement | null>(null);
+
+  return (
+    <AutofixDrawerContainer>
+      <AutofixDrawerHeader>
+        <NavigationCrumbs
+          crumbs={[
+            {
+              label: (
+                <CrumbContainer>
+                  <ProjectAvatar project={project} />
+                  <ShortId>{group.shortId}</ShortId>
+                </CrumbContainer>
+              ),
+            },
+            {label: getShortEventId(event.id)},
+            {label: t('Autofix')},
+          ]}
+        />
+      </AutofixDrawerHeader>
+      <AutofixNavigator>
+        <Header>{t('Autofix')}</Header>
+        {autofixData && (
+          <ButtonBar gap={1}>
+            <AutofixFeedback />
+            <Button
+              size="xs"
+              onClick={reset}
+              title={
+                autofixData.created_at
+                  ? `Last run at ${autofixData.created_at.split('T')[0]}`
```

Comment:
not sure how important but this doesn't respect user timezone. date will be in utc
```suggestion
                  ? `Last run at ${autofixData.created_at.split('T')[0]}`
```